cmd/api/src/daemons/datapipe/ingest.go

```diff
@@ -60,11 +58,11 @@ func (s *Daemon) IngestAzureData(batch graph.Batch, converted ConvertedAzureData
 }
 
 func (s *Daemon) IngestWrapper(batch graph.Batch, wrapper DataWrapper) error {
-	// TODO: Cleanup #ADCSFeatureFlag after full launch.
-	adcsFlag, err := s.db.GetFlagByKey(appcfg.FeatureAdcs)
-	if err != nil {
-		return fmt.Errorf("error getting ADCS feature flag: %w", err)
-	}
+	//Leaving this code here for re-enabling in future ADCS work
+	//adcsFlag, err := s.db.GetFlagByKey(appcfg.FeatureAdcs)
+	//if err != nil {
+	//	return fmt.Errorf("error getting ADCS feature flag: %w", err)
+	//}
```

Comment:
Recommend ripping this since it's both trivial and Git will keep track of it for us